kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -70,8 +69,247 @@
+              <KGridItem :layout12="{ span: 4 }">
+                <button class="icon-container remove-button-style">
+                  <KIcon
+                    class="reduce-chervon-spacing"
+                    icon="chevronDown"
+                  />
+                  <KIcon
+                    class="reduce-chervon-spacing"
+                    icon="chevronUp"
+                  />
+                </button>
+              </KGridItem>
+
+              <KGridItem
+                :layout12="{ span: 4 }"
+              >
+
+                <KIconButton
+                  class="icon-size"
+                  icon="refresh"
+                />
+              </KGridItem>
+
+              <KGridItem
+                :layout12="{ span: 4 }"
+              >
+                <KIconButton
+                  class="icon-size"
+                  icon="trash"
+                />
+              </KGridItem>
+            </KGrid>
+          </div>
+        </KGridItem>
+
+      </KGrid>
+      <DragContainer
+        :items="placeholderList"
+      >
+        <AccordionContainer>
+          <template
+            #default="{ isItemExpanded, toggleItemState }"
+          >
+            <Draggable
+              v-for="(item,index) in placeholderList"
+              :key="item.id"
+              tabindex="-1"
+            >
+              <AccordionItem
+                :id="item.id"
+                :key="item.id"
+                :items="placeholderList"
+                :title="item.title"
+                :expanded="isItemExpanded(item.id)"
+              >
+                <template
+                  #heading="{ title }"
+                  :accordionToggle="onAccordionToggle(item.id)"
+                >
+                  <button
+                    tabindex="-1"
+                    aria-expanded="false"
+                    aria-label="toggle-button"
+                    class="remove-button-style"
+                    @click="toggleItemState(item.id)"
+                  >
+                    <div
+                      class="flex-div"
+                    >
+                      <div
+                        class="left-column-alignment-style"
+                      >
+                        <Draggable
+                          v-if="!isItemExpanded(item.id)"
+                        >
+                          <DragHandle>
+                            <DragSortWidget
+                              class="drag-icon sort-widget"
+                              :moveUpText="$tr('upLabel', { name: item.name })"
+                              :moveDownText="$tr('downLabel', { name: item.name })"
+                              :isFirst="index === 0"
+                              :isLast="index === placeholderList.length - 1"
+                              @moveUp="shiftOne(index, -1)"
+                              @moveDown="shiftOne(index, +1)"
+                              @click.prevent="toggleItemState(item.id)"
```

Comment:
I think this is the source of one issue noted by @radinamatic -- this should be triggering the updating of the order of the items, but not toggling the item state. The "toggleItemState" should isntead be on a button that wraps the whole row (I think the button below around line 210 or so).

See the `RearrangeChannelsPage` for an example; there the `@sort` event is emitted by the `DragContainer` where the new order of the items is handled.